add_to_app/multiple_flutters/multiple_flutters_android/app/src/main/java/dev/flutter/multipleflutters/SingleFlutterActivity.kt

```diff
@@ -0,0 +1,37 @@
+package dev.flutter.multipleflutters
+
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import io.flutter.embedding.android.FlutterActivity
+import io.flutter.embedding.engine.FlutterEngine
+
+/**
+ * This is an Activity that displays one instance of Flutter.
+ *
+ * EngineBindings is used to bridge communication between the Flutter instance and the DataModel.
+ */
+class SingleFlutterActivity : FlutterActivity(), EngineBindingsDelegate {
+    private val engineBindings : EngineBindings by lazy {
```

Comment:
lateinit

Reply:
mentioned above my these should still be lazy

Reply:
ack